lib/std/Build/Step/Run.zig

```diff
@@ -240,6 +242,40 @@ pub fn addPrefixedDirectoryArg(self: *Run, prefix: []const u8, directory_source:
     directory_source.addStepDependencies(&self.step);
 }
 
+/// Add a standard `--dep-file=<deps.d>` argument for the child process to fill
+/// with its discovered additional dependencies.
+pub fn addDepFileOutputArg(self: *Run) std.Build.LazyPath {
+    return self.addPrefixedDepFileOutputArg("--dep-file=");
+}
+
+/// Add a dependencies file argument for the child process to fill with its
+/// discovered additional dependencies. Same as addDepFileOutputArg but you can
+/// specify the `flag` to use.
+/// `flag` supports both `"--some-flag="` (single argument) and `"--some-flag"`
+/// (two arguments) forms.
+pub fn addPrefixedDepFileOutputArg(self: *Run, flag: []const u8) std.Build.LazyPath {
+    const b = self.step.owner;
+
+    // Handle both `--dep-file=<deps.d>` and `--dep-file <deps.d>`
+    const two_args = !std.mem.endsWith(u8, flag, &.{'='});
```

Comment:
Don't do this. It's too late to discover the user's intention. In this function, there is a single argument that is prefixed with `flag` (which should be renamed to `prefix`).